### ISSUE TITLE
Step Templates created on old versions of Octopus with Package References keep getting errors when importing into new.

### ISSUE THREAD

**Issue by OctopusBob** [OctopusDeploy/Issues#5432]
# Prerequisites

- [x] I have verified the problem exists in the latest version
- [x] I have searched [open](https://github.com/OctopusDeploy/Issues/issues) and [closed](https://github.com/OctopusDeploy/Issues/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed) issues to make sure it isn't already reported
- [x] I have written a descriptive issue title
- [x] I have linked the original source of this report
- [x] I have tagged the issue appropriately (area/*, kind/bug, tag/regression?)

# The bug

In 2018.9.x we added the "ID" field to all packages in step templates.  If I attempt to update a step template without that ID field in 2018.8.x+ this error is thrown:

![image](https://user-images.githubusercontent.com/38642825/55237955-51a93780-5201-11e9-949a-5aeafb0cc87f.png)

In newer editions of Octopus Deploy this error is shown:

![image](https://user-images.githubusercontent.com/38642825/55237990-6685cb00-5201-11e9-8240-732ccdeb5f23.png)

When the ID is added to the package (version 4 of the step template) that fixes it...temporarily.  When version 5 of the step template is created we are back to the same thing.  When we look at the step template itself in the UI went from this:

![image](https://user-images.githubusercontent.com/38642825/55238099-acdb2a00-5201-11e9-9ddc-c94b5720cdfb.png)

To this:

![image](https://user-images.githubusercontent.com/38642825/55238114-b6649200-5201-11e9-8841-147599d15733.png)

This is happening for step templates based on "run a script" and "deploy a package" steps.  This only happens imported in from the community library.

## What I expected to happen

For the package reference not to be lost.

## Steps to reproduce

1. Create a step template with a package reference using a version less than or equal to 2018.7.  Export that step template.
2. Attempt to import it into 2019.8 or higher (whichever version we added IDs to package reference)
3. Error!

## Affected versions

**Octopus Server:** 2018.8.x+

## Workarounds

Manually add that ID, and hope that it works?

## Links

This was reported by a TAM customer.

**Comment by michaelnoonan** [comment]
To clarify, this has nothing to do with `Octopus.Migrator.exe import`? It is the Library > Step Templates > Import from JSON?

**Comment by OctopusBob** [comment]
That is correct. It has nothing to do with the migrator. It is using the UI to import step templates. From JSON or from the community. 

Bob
